src/learn.jl

```diff
@@ -63,6 +63,16 @@ Log a value `value` to `field`.
 """
 log_value!(logger, field::AbstractString, value)
 
+
+"""
+    log_array!(logger::Any, field::AbstractString, value)
+
+Log an array `value` to `field`.
+
+Defaults to `log_value!(logger, mean(value))`.
+"""
+log_array!(logger::Any, field::AbstractString, value) = log_value!(logger, field, mean(value))
```

Comment:
```suggestion
log_array!(logger::Any, field::AbstractString, array) = log_value!(logger, field, mean(array))
```
...for ease of reading, I find it nice when the param name matches the function name? your choice!